src/styles/blocks/card.css

```diff
@@ -0,0 +1,88 @@
+/* #region card */
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__image {
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+  display: block;
+}
+
+.product-card__name {
+  margin-bottom: 4px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060b35;
+}
+
+.product-card__code {
+  margin-bottom: 16px;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  display: block;
+  color: #616070;
+}
+
+.product-card__rating {
+  margin-bottom: 24px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.product-card__reviews {
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.product-card__price {
+  margin-bottom: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.product-card__price-name {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.product-card__price-value {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+  color: #060b35;
+}
+
+.product-card__purchase {
+  box-sizing: border-box;
+  width: 166px;
```

Comment:
Why do you need to hardcode these values? What if width of card will be changed?